encoding/kzg/verifier/verifier.go

```diff
@@ -329,3 +329,45 @@ func PairingsVerify(a1 *bn254.G1Affine, a2 *bn254.G2Affine, b1 *bn254.G1Affine,
 
 	return nil
 }
+
+// GenerateBlobCommitment computes a kzg-bn254 commitment of blob data using SRS
+func (v *Verifier) GenerateBlobCommitment(blob []byte) (*encoding.G1Commitment, error) {
```

Comment:
does this need to be public?

Reply:
I don't have any particular external use cases in mind. My tendency would be to keep it public, since it's a general purpose method to compute commitment from bytes, with likelihood of being used elsewhere in the future. But I'd have no strong objection to making it private

Reply:
We can rework the API later but I find it a bit weird that this method is on a strict called Verifier. I might want to compute a kzg commitment without verifying anything.

Reply:
I'll make it private for now, and we can reconsider moving somewhere else more suitable, as future use cases present themselves

Reply:
I see your point. We could create a new type `Committer`, which would accept a kzg configuration, and just produce the commitments.